tests/test_venue_configuration.py

```diff
@@ -0,0 +1,420 @@
+        expdate = openreview.tools.datetime_millis(now + datetime.timedelta(minutes=28))        
+
+        edit = pc_client.post_invitation_edit(
+            invitations='ICLR.cc/2025/Conference/-/Stage',
+            content={
+                'stage_name': { 'value': 'Confidential_Comment' },
+                'stage_type': { 'value': 'Official_Comment' },
+                'activation_date': { 'value': cdate },
+                'expiration_date': { 'value': expdate }
+            }
+        )
+
+        helpers.await_queue_edit(openreview_client, edit_id=edit['id'], count=1)
+
+        assert pc_client.get_invitation('ICLR.cc/2025/Conference/-/Confidential_Comment')
+
+        helpers.await_queue_edit(openreview_client, edit_id='ICLR.cc/2025/Conference/-/Confidential_Comment-0-1', count=1)
+
+        assert pc_client.get_invitation('ICLR.cc/2025/Conference/Submission1/-/Confidential_Comment')
+
+        submissions = openreview_client.get_notes(invitation='ICLR.cc/2025/Conference/-/Submission', sort='number:asc')
+        pc_client.post_note_edit(invitation=f'ICLR.cc/2025/Conference/Submission1/-/Confidential_Comment',
+            signatures=['ICLR.cc/2025/Conference/Program_Chairs'],
+            note=openreview.api.Note(
+                replyto=submissions[0].id,
+                content={
+                    'comment': { 'value': 'this is a comment' }
+                }
+            ))
+
+    def test_metareview_stage(self, openreview_client, test_client, helpers):
+
+        pc_client = openreview.api.OpenReviewClient(username='sherry@iclr.cc', password=helpers.strong_password)
+
+        now = datetime.datetime.now()
+        cdate = openreview.tools.datetime_millis(now - datetime.timedelta(minutes=1))
+        duedate = openreview.tools.datetime_millis(now + datetime.timedelta(minutes=30))
+        expdate = openreview.tools.datetime_millis(now + datetime.timedelta(hours=1))
+
+        edit = pc_client.post_invitation_edit(
+            invitations='openreview.net/Support/-/Meta_Review_Template',
+            signatures=['~ProgramChair_ICLR1'],
+            content={
+                'venue_id': { 'value': 'ICLR.cc/2025/Conference' },
+                'stage_name': { 'value': 'Meta_Review' },
+                'activation_date': { 'value': cdate },
+                'due_date': { 'value': duedate },
+                'expiration_date': { 'value': expdate }
+            }
+        )
+        helpers.await_queue_edit(openreview_client, edit_id=edit['id'], count=1)
+
+        assert pc_client.get_invitation('ICLR.cc/2025/Conference/-/Meta_Review')
+        assert pc_client.get_invitation('ICLR.cc/2025/Conference/-/Meta_Review/Deadlines')
+        assert pc_client.get_invitation('ICLR.cc/2025/Conference/-/Meta_Review/Form_Fields')
+        assert pc_client.get_invitation('ICLR.cc/2025/Conference/-/Meta_Review/Readers')
+
+        invitations = pc_client.get_invitations(invitation='ICLR.cc/2025/Conference/-/Meta_Review')
```

Comment:
@melisabok any ideas whay this line works but line#419 does not? For some reason, I can't get official review invitations by invitation even though I know they exist

Reply:
> I think you need to wait until the date process function finishes to check the count: "ICLR.cc/2025/Conference/-/Meta_Review-0-1"

I do this in line 338 and I still can't get official review invitations. I have no issue getting metareview invitations. The only difference is that the Official Review invitation has been edited by the subinvitations, not sure if this has anything to do.

Reply:
Weird, let me check again, in my localhost it failed getting the meta review invitations.

Reply:
I realize I forgot to check `ICLR.cc/2025/Conference/-/Meta_Review-0-1` had finished but I am actually not having issues with the metareviews

Reply:
this is probably because your machine is too fast!, which line is your test failing?

Reply:
the code that is commented out in lines 340 and 427, when I try to get the review invitations

Reply:
I pushed a change that should fix the assertions:

- These invitations were expired, we can use `expired=True` or change the expdate. I changed the dates.
- There was an await queue missing to wait the meta review invitations were created.

We still have the issue of using utcnow that I need to fix in another PR